Formula/jdtls.rb

```diff
@@ -0,0 +1,63 @@
+class Jdtls < Formula
+  desc "Java language specific implementation of the Language Server Protocol"
+  homepage "https://github.com/eclipse/eclipse.jdt.ls"
+  url "https://download.eclipse.org/jdtls/milestones/1.7.0/jdt-language-server-1.7.0-202112161541.tar.gz"
+  sha256 "2f0c28dfec317a268ec44904420657181b43a7ba2a32f0bf788ea388dacb8552"
+  license "EPL-2.0"
+
+  depends_on "openjdk@11"
+
+  def install
+    etc.install "config_mac" => "jdtls"
+    libexec.install "features"
+    libexec.install "plugins"
+
+    (buildpath/"jdtls-wrapper").write <<~BASH
+      #!/bin/bash
+      # copy the configuration folder to tmp to be writable
+      tmp_dir="$(mktemp -d /tmp/jdtls.XXXXX)"
+      cp -R #{etc}/jdtls "${tmp_dir}"
+      # and ensure that it is removed on exit
+      trap "{ rm -rf ${tmp_dir}; }" EXIT
```

Comment:
If it can't work without this I don't think it's suitable for homebrew.

See https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-heavy-manual-prepost-install-intervention

Reply:
I found a solution that works with `write_jar_script` and with a read-only configuration directory (with the aid of an [Eclipse bug report comment](https://bugs.eclipse.org/bugs/show_bug.cgi?id=241730#c6)).

The list of Java properties is i bit longer but [other tools](https://github.com/Homebrew/homebrew-core/blob/e2ceceeaaa72e5353d93db035edc1a98134618c1/Formula/google-java-format.rb) also have longer lists.